src/box/sql/func.c

```diff
@@ -716,7 +732,7 @@ func_position_characters(struct sql_context *ctx, int argc,
 		if (res != NULL) 
 			pos = res - str + 1;
 	} else {
-		pos = kmp_single(ctx->coll, str, str_size, key, key_size);
+		pos = kmp_first(ctx->coll, str, str_size, key, key_size);
 		if (pos == -1) {
 			diag_set(OutOfMemory, 0, "sql_cmp_calloc", "kmp");	//TODO change 0 to smth
```

Comment:
I don't see a pleasant way to pass the desired allocation capacity here. Is it important?